uber/templates/registration/register.html

```diff
@@ -1,131 +1,64 @@
-<!DOCTYPE HTML>
```

Comment:
Ah, sorry for being unclear.

If we just put ``{% if %}`` checks in the template in the core ``regform.html``, that would cause two problems:

1) We still want those fields to appear on the admin form.  Therefore, removing them seems more sensible to put in the form(s) where we want them removed.

2) We want an event-specific plugin to be able to easily override the removal.  If we wrap the blocks inside an ``{% if %}`` check, then the fields aren't ever there in the first place for the plugin's Javascript code to affect.  This way the HTML is still there and we can make a simple JS change to keep the fields around.